doc/src/guides/solving/solve-equation-algebraically.md

```diff
@@ -0,0 +1,279 @@
+# Solve an equation algebraically
+
+Use SymPy to solve an equation algebraically (symbolically). For example, solving $x^2 = y$ yields $x \in \{-\sqrt{y},\sqrt{y}\}$.
```

Comment:
This should immediately have links to other pages e.g. for when you have more than one equation or want to find integer solutions etc.

Reply:
Linked to solving main page.